website/docs/r/vpc_flow_log.html.markdown

```diff
@@ -0,0 +1,88 @@
+
+---
+subcategory: "VPC"
+layout: "alicloud"
+page_title: "Alicloud: alicloud_vpc_flow_log"
+sidebar_current: "docs-alicloud-resource-vpc-flow-log"
+description: |-
+  Provides a Alicloud VPC Flow Log resource.
+---
+
+# alicloud\_vpc\_flow\_log
+
+Provides a VPC Flow Log resource.
+
+For information about VPC Flow log and how to use it, see [Flow log overview](https://www.alibabacloud.com/help/doc-detail/127150.htm).
+
+-> **NOTE:** Available in v1.116.0+
+
+-> **NOTE:** While it uses `alicloud_vpc_flow_log` to build a vpc flow log resource, it will be active by default.
```

Comment:
If it defaults to active, the status should have a default Active to control it.